test/functional/page_objects/visualize_page.ts

```diff
@@ -47,6 +47,14 @@ export function VisualizePageProvider({ getService, getPageObjects }: FtrProvide
       await listingTable.clickNewButton('createVisualizationPromptButton');
     }
 
+    public async clickAggBasedVisualizations() {
+      await testSubjects.click('visGroupAggBasedExploreLink');
+    }
+
+    public async goBacktoGroups() {
```

Comment:
nit: common mistake.  we don't need to add `async/await` here cause in code below anyway add `await` operator

![image](https://user-images.githubusercontent.com/20072247/96414484-d8fc3880-11f5-11eb-8239-ab9e6c0526af.png)